_sass/_header.scss

```diff
@@ -2,16 +2,45 @@ header.site-header{
 
     background-color: $dark-grey;
     background-image: url($header_background_url);
+    background-size: cover;
     padding: 20px 0 18px 0;
 
+    .wrapper {
+      align-items: center;
+      display: flex;
+      justify-content: space-between;
+
+      @media (max-width: 767px) {
+        flex-direction: column;
+      }
+    }
+
     .logo {
 
         a {
-            float: left;
             display: inline-block;
             color: $white;
         }
 
+        @media (max-width: 767px) {
+          margin-bottom: 8px;
+        }
+    }
+
+    .subscribe {
+      padding: 10px 12px;
+      background-color: #2273c3;
+      border-radius: 4px;
+      color: white;
+      font-size: 10px;
+      font-weight: 400;
+      text-shadow: 0 1px 0 rgba(0, 0, 0, 0.1);
+      text-transform: uppercase;
+      letter-spacing: 2px;
+
+      &:hover {
```

Comment:
after checking out in the wild i am not sure it is picking up on the hover effect here. If I may make a suggestion you could do this instead  just outside of the .subscribe block target the hover explicitly with a color hex from patterns 🤙 

.subscribe:hover {
   background-color: #1c5c9c;
}

Reply:
Lemme know if that works for ya! 😄

Reply:
The url in the description is wrong.  Try this one: https://buildkite.yden.us/jekyll-theme-guardian/7/_site/index.html

Reply:
...lol Well it works 😆